src/memex/storage.py

```diff
@@ -120,15 +120,17 @@ def create_annotation(request, data):
     annotation = models.Annotation(**data)
     annotation.created = created
     annotation.updated = updated
-    request.db.add(annotation)
 
-    models.update_document_metadata(
+    document = models.update_document_metadata(
         request.db,
         annotation.target_uri,
         document_meta_dicts,
         document_uri_dicts,
         created=created,
         updated=updated)
+    annotation.document_id = document.id
```

Comment:
Maybe add a `FIXME` here to come back and update this to just do `annotation.document = document` once we've updated the relationship property?